types/signer.go

```diff
@@ -32,13 +34,22 @@ type BeaconSigner interface {
 	IsBeaconBlockSlashable(pk []byte, slot spec.Slot) error
 }
 
-// SSVSigner used for all SSV specific signing
-type SSVSigner interface {
+// SSVOperatorSigner used for to sign protocol messages
+type SSVOperatorSigner interface {
+	SignSSVMessage(data []byte, pk []byte) ([]byte, error)
+	// AddSSVOperatorKey saves an SSV operator key
+	AddSSVOperatorKey(sk *rsa.PrivateKey) error
+	// RemoveSSVOperatorKey removes an SSV operator key
+	RemoveSSVOperatorKey(pubKey string)
```

Comment:
IMO these are redundant, don't make us satisfy this interface (😆); There is always just a single operator key per instance, so there's never a need to add and remove them.